tests/imresh/io/testReadWrite.cpp

```diff
@@ -102,8 +102,8 @@ namespace io
                 for ( auto i = 0; i < nElements; ++i )
                     pData[i] = (float) rand() / RAND_MAX;
 
-                /* note that this call deletes the pointer @todo: move delete[] inside taskQueue.cu */
-                writeOutPNG( pData, ImageDim{ Nx, Ny }, tmpFileName );
+                writeOutAndFreePNG( pData, ImageDim{ Nx, Ny }, tmpFileName );
+                delete[] pData;
```

Comment:
But now I don't get why you'd have to call `delete[]` after you've called a `writeOutAndFree*` function.